bin/sandbox

```diff
@@ -115,7 +115,7 @@ bundle exec rails generate solidus:install \
   --frontend=none \
   $@
 
-bundle exec app:template LOCATION="$PWD/../template.rb"
+bundle exec rails app:template LOCATION="$PWD/../template.rb"
```

Comment:
Shouldn't this go in the previous commit?

Reply:
Yes, sure, I was just checking if it was fixing the CI, but when I saw it was green I eagerly set the PR as ready for review 😅